### PR TITLE
Initial PR for #73

### LINKED ISSUE #73: Enhancement ideas for post-install validation script
With #71 , we introduced an initial form of post-install validation for openshift. We also captured some ideas for future enhancements, which I will move to this issue for tracking.

- Fine tune the execution through command line parameters
  - Ability to skip certain tests
  - Set the expected values for certain tests (ie: number of masters)
    - We can look into the addoption functionality. https://docs.pytest.org/en/latest/example/simple.html

### PR DESCRIPTION
**Pull request by aizuddin85**
@etsauer this initial PR to resolve #73 . Let me know if I heading the right way with this PR then I can continue the enhancement.

### CODE REVIEW

**Comment by etsauer** [comment]
hey @aizuddin85 this looks neat! Can you add some instructions in the readme on how to use the flags so I can test the options?

File: validation/pytest.ini [hunk 1]
```diff
@@ -0,0 +1,9 @@
+[pytest]
+# Run all tests
+#addopts = --etcd-node-count=3 --router-node-count=3 --registry-pod-count=3 --master-node-count=3
```
Comment (etsauer) [inline validation/pytest.ini]: I think we should default to run all tests.

**Comment by aizuddin85** [comment]
@etsauer update README.md for example to run. Agree that by default it should run everything with ability to skip certain tests.

**Comment by etsauer** [comment]
@aizuddin85 can you make that change to the .ini file to run everything by default please?

**Comment by aizuddin85** [comment]
@etsauer did several changes and update pytest.ini to run all as default.

**Comment by aizuddin85** [comment]
Closing this one. Create new PR #81

**Comment by etsauer** [comment]
@aizuddin85 hey, I merged #81. In the future, you don't need to close PRs. You can just push new commits to the same branch to address comments. That way we maintain the conversations we had around the change.

**Comment by aizuddin85** [comment]
@etsauer thanks for accepting the PR. Reason behind I closed this PR due to too many merge conflicts that I wrongly introduced when I fetched from upstream. Still learning the best way to fix multiple merge conflicts. :-)

**Pull request closed without merging**